services/web-app/sst-env.d.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
```

Comment:
Can you ignore this file in your eslint rules? Maybe ignore all *.d.ts?

Reply:
Good shout